gremlin-server/pom.xml

```diff
@@ -148,6 +149,30 @@ limitations under the License.
     </build>
     <profiles>
         <profile>
+            <id>linux</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+                <property>
+                    <name>EPOLL</name>
```

Comment:
good point - i might ultimately adjust naming/docs and such once we merge.  @anagav i can handle that on my end.  @pluradj is that cool for you for purpose of voting?